static-assets/components/cstudio-contextual-nav/wcm-site-dropdown-mods/wcm-root-folder.js

```diff
@@ -902,31 +902,36 @@
       /**
        * draw a canned search element
        */
-      drawCannedSearch: function (searchConfig, root) {
+       drawCannedSearch: function (searchConfig, root) {
         var treeNode = null;
         var searchId = CStudioAuthoring.Utils.generateUUID();
         var newId = CStudioAuthoring.Utils.generateUUID();
         if (searchConfig.newPath && !window.pasteFlag) {
-          var label =
+          if (!(/^<a.*>.*<\/a><a.*>\+add<\/a>/.test(searchConfig.label))) {
```

Comment:
It looks like you're testing to see if the anchor is already there but the regex hardcodes the English label. 
Is that button not internationalized? What if you're running a different language?

Reply:
Hi @rart 
Yes, seem like the `+add` button is not internationalized.
Originally, it is in this file:

```
var label =
            "<a style='display: inline; padding-right:5px;' id='ID' href='#' class='canned-search-el'>LABEL</a><a style='display: inline; border-left: 1px solid grey; padding-left: 5px;' id='NEWID' href='#'>+add</a>";
```
![add_btn_canned_search](https://user-images.githubusercontent.com/2996543/155321791-d803c40b-d7b2-425e-8782-00af079960b6.png)